### ISSUE TITLE
Not detecting the main games of "Demon Turf: Queens Edition" at the Installed Game tab

### ISSUE THREAD

**Issue by KwaiBird** [Raicuparta/rai-pal#225]
Hi,

Because [Demon Turf: Queens Edition](https://store.steampowered.com/app/1325900/Demon_Turf_Queens_Edition/) uses Unity as the launcher of the game variants, Rai Pal detects the launcher as the main game at the Installed Game tab. 
The app detects "Demon Turf" and "Demon Turf: Neon Splash", but the former is linked to the launcher.
The execution files are at "Demon Turf" root directory, "Demon Turf\Demon Turf_Data\Demon Turf", "Demon Turf\Demon Turf_Data\Demon Turf Neon Splash" and "Demon Turf\Demon Turf_Data\Demon Turf Tower".
Directory tree:[tree.txt](https://github.com/user-attachments/files/16118071/tree.txt)

Rai Pal v.0.12.1

**Comment by Raicuparta** [comment]
If you drag the correct exe manually over to Rai Pal, does it work correctly?

There's only so much Rai Pal can do to guess this stuff, and trying to do deeper analysis on every game to figure out whether a particular unity app is a game or a launcher or whatever would make the scanning process too slow (it's already pretty slow tbh).

I'll still have a look at this to see if anything can be done about it though.

**Comment by KwaiBird** [comment]
Yes. If I do that manually and install UUVR and the config manager on the app, they work fine.
I appreciate about further checking, and I understand analyzing the somewhat tricky structure may not pay off.

**Comment by Raicuparta** [comment]
Closing all issues about individual games, we'll tack in #315 instead.
